src/Processors/Formats/Impl/JSONColumnsBlockOutputFormatBase.h

```diff
@@ -20,6 +20,13 @@ class JSONColumnsBlockOutputFormatBase : public OutputFormatWithUTF8ValidationAd
 
     String getName() const override { return "JSONColumnsBlockOutputFormatBase"; }
 
+    String getContentType() const override
+    {
+        if (!format_settings.json.content_type.empty())
+            return format_settings.json.content_type;
+        return OutputFormatWithUTF8ValidationAdaptor::getContentType();
```

Comment:
```suggestion
        return "application/json; charset=UTF-8";
```

It was actually a mistake that this class didn't have implementation of `getContentType`